src/portal/lib/src/tag/tag.component.ts

```diff
@@ -776,4 +767,26 @@ export class TagComponent implements OnInit, AfterViewInit {
   onCpError($event: any): void {
     this.copyInput.setPullCommendShow();
   }
+  getProjectScanner(): void {
+    this.hasEnabledScanner = false;
+    this.scanBtnState = ClrLoadingState.LOADING;
+    this.http.get(`/api/projects/${this.projectId}/scanner`)
+        .pipe(map(response => response as any))
+        .pipe(catchError(error => observableThrowError(error)))
+        .subscribe(response => {
+          if (response && "{}" !== JSON.stringify(response) && !response.disable
+          && response.health) {
+            this.hasEnabledScanner = true;
+          }
+          this.scanBtnState = ClrLoadingState.SUCCESS;
+        }, error => {
+          this.scanBtnState = ClrLoadingState.ERROR;
+        });
+  }
+  handleScanOverview(scanOverview: any) {
+    if (scanOverview) {
+      return scanOverview['application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0'];
```

Comment:
Seems this 'application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' is hard code everywhere. Eventhough a const defined in a component. For other components still need to hard code it. It is better to define it in a const file and export it.

Reply:
fixed